pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1702,6 +1702,65 @@ func PublishVolumeToZedCloud(ctx *zedagentContext, uuid string,
 	}
 }
 
+// PublishBlobInfoToZedCloud is called per change, hence needs to try over all management ports
+// When blob Status is nil it means a delete and we send a message
+// containing only the UUID to inform zedcloud about the delete.
+func PublishBlobInfoToZedCloud(ctx *zedagentContext, blobSha string, blobStatus *types.BlobStatus, iteration int) {
+	log.Infof("PublishBlobInfoToZedCloud blobSha %v", blobSha)
+	var ReportInfo = &info.ZInfoMsg{}
+
+	contentType := new(info.ZInfoTypes)
+	*contentType = info.ZInfoTypes_ZiBlobList
+	ReportInfo.Ztype = *contentType
+	ReportInfo.DevId = *proto.String(zcdevUUID.String())
+	ReportInfo.AtTimeStamp = ptypes.TimestampNow()
+
+	ReportBlobInfoList := new(info.ZInfoBlobList)
+
+	ReportBlobInfo := new(info.ZInfoBlob)
+
+	ReportBlobInfo.Sha256 = blobSha
+	ReportBlobInfo.State = info.ZSwState_HALTED
```

Comment:
Or leave it at zero? Makes sense to check how the controller wants to handle deletes.

Reply:
Controller does not handle delete as far as I saw. Will raise a PR on the controller.

Reply:
ok